server/repository/experiments.go

```diff
@@ -69,3 +70,19 @@ func (repo *Experiments) GetAll() ([]*model.Experiment, error) {
 
 	return results, nil
 }
+
+// Create experiment model in database
```

Comment:
I'd mention that the new ID is also set.

`// Create stores an Experiment in database. On success the assigned ID is set`